### ISSUE TITLE
Configuring the database to run the example server

### ISSUE THREAD

**Issue by kmpartner** [stellar/django-polaris#574]
### What version are you using?
polaris 2.1.0

### What did you do?
I tried to use Polaris 2.1.0 deployed in kubernetes.
sep-24 deposit did not work in demo-wallet using I created test token (when I used Polaris 1.6.4, it worked).

When I tried sep-24 deposit, Error message show 
DoesNotExist at /sep24/transactions/deposit/webapp/submit
OffChainAssetExtra matching query does not exist.

How can I solve these problems?
in Polaris 2.1.0, admin UI has changed from 1.6.4, and I am not sure configurations (Delivery methods, Exchange pairs ...) are correctly configured.

**Comment by JakeUrban** [comment]
Hi @kmpartner 

First I would upgrade to the latest patch release, 2.1.2.

This is also not a bug, it just sounds like you don't have the necessary database entries to run the example server successfully. Our example server demonstrates how off-chain assets can be exchanged for different on-chain assets, so the example server has some new models.

If you want to run the example server, you need to create an USD `OffChainAsset` record, as well as a `OffChainAssetExtra` record. Then, you can create an `ExchangePair` record for USD and SRT. Finally, you'll want to add at least one `DeliveryMethod` record associated with the USD off-chain asset record.

**Comment by kmpartner** [comment]
Thank you for your response,
I am not sure what is OffChainAssetExtra, and where can I save OffChainAssetExtra?
Is it in OffChainAsset?

**Comment by JakeUrban** [comment]
`OffChainAssetExtra` lives in the `server.models` module. They're custom models implemented in the example app vs. Polaris' built-in models.

**Comment by kmpartner** [comment]
Thank you for your response.
After using hardcoded OffCainAssetExtra values in code, OffChainAssetExtra error message was gone.
But after confirm transaction button is clicked, Only '403 forbidden' is shown on the screen.
How can I solve this problem?
Is this forbidden message caused by using SRT of https://testanchor.stellar.org ?

**Comment by kmpartner** [comment]
After changing OffChainAssetExtra values and environment variables.
'fobidden' message disappeared, but sep-24 deposit did not work.

After confirm transaction button is clicked, Screen's Transaction Status did not change from 'waiting on the user to transfer funds'.

transaction status did not change from 'pending_user_transfer_start', and anchor's log keep showing,
'server                           | 2021-12-27 17:51:51,964 - INFO: "GET /sep24/transaction?id=8ccd49c1-3bed-4562-af67-afa731f33ec7 HTTP/1.1" 200 860'

How can I solve this problem?

Transaction in browser's network in browser's developer tool showed below,
transaction: {id: "8ccd49c1-3bed-4562-af67-afa731f33ec7", kind: "deposit", status: "pending_user_transfer_start",…}
amount_fee: "0.00"
amount_fee_asset: "stellar:SRT:GCDNJUBQSX7AJWLJACMJ7I4BC3Z47BQUTMHEICZLE6MU4KQBRYG5JY6B"
amount_in: "10.00"
amount_in_asset: "iso4217:USD"
amount_out: "16.67"
amount_out_asset: "stellar:TEST6:GAJPUFR7JP5OR464MARH3IBFYB2PTYOXT7X2C75DB5273IMCRBAA7SPW"
claimable_balance_id: null
completed_at: null
deposit_memo: null
deposit_memo_type: "hash"
external_transaction_id: null
from: null
id: "8ccd49c1-3bed-4562-af67-afa731f33ec7"
kind: "deposit"
message: "waiting on the user to transfer funds"
more_info_url: "http://localhost:8000/sep24/transaction/more_info?id=8ccd49c1-3bed-4562-af67-afa731f33ec7"
refunded: false
started_at: "2021-12-28T01:49:48.756438Z"
status: "pending_user_transfer_start"
status_eta: null
stellar_transaction_id: null
to: "GDFHUFLDWEPYDKOJA2NQ75XYJVA7L4CDHNZNYXNLGUJOA35UA4JU7DUQ"

**Comment by yuriescl** [comment]
You must implement `DepositIntegration.poll_pending_deposits` in order to tell Polaris which deposits were received from the user.
You also need to run `process_pending_deposits`. Are you getting any errors when running it?

**Comment by kmpartner** [comment]
How can I implement DepositIntegration.poll_pending_deposits?
process_pending_deposits command exists in inpolaris/polaris/management/commands folder, but there is no command for poll_pending_deposits.

**Comment by yuriescl** [comment]
Sorry I mistyped, it's actually [RailsIntegration.poll_pending_deposits](https://django-polaris.readthedocs.io/en/stable/sep6_and_sep24/index.html#polaris.integrations.RailsIntegration.poll_pending_deposits).
When you run `python manage.py process_pending_deposits`, Polaris will run `RailsIntegration.poll_pending_deposits`. In there you need to implement your own logic to determine which deposits were received from the user. Polaris will update the transaction status depending on the transactions you return from that function.
From the function docs:
_This function should poll the appropriate financial entity for the state of all pending_deposits and return the ones that have externally completed, meaning the off-chain funds are available in the anchor’s account_
...
_For every transaction that is returned, Polaris will submit it to the Stellar network_

**Comment by yuriescl** [comment]
Polaris has an example implementation at https://github.com/stellar/django-polaris/blob/master/example/server/integrations/rails.py#L25

**Comment by yuriescl** [comment]
I would review the settings first, check `STELLAR_NETWORK_PASSPHRASE` and `HORIZON_URI` variables, they should point to testnet/public depending on your environment.

**Comment by kmpartner** [comment]
Thank you for your response,
When deployed Polaris using kubernetes in the cloud environment, sep-24 deposit for some assets succeeded. But when trying sep-24 deposit locally using docker-compose, "tx_bad_auth" error in **process_pending_deposits** still remains because of unknown reasons.

Why is there no place for OffChainAssetExtra creation on the Polaris admin UI?

**Comment by kmpartner** [comment]
By the way, I tested sep-31 send locally and that works using demo-wallet.
I have questions about sep-38 integration in sep-31.

How can I implement sep-38 in sep-31 send?
I don't find to fill destination_asset in sep-31 sending modal in demo-wallet.

I tested sep-38 endpoints using locally running Polaris as well. 
When I requested /prices endpoint, prices are returned but returned prices seem to be a random number. Where can I find the prices of sep-38 get request in Polaris code?
